### DISCUSSION TITLE
Disable management network/interfaces

### DISCUSSION THREAD

**Discussion by karampok** [srl-labs/containerlab#2008] [General] [answered]
Hi, is there a possibility to disable all the way the management network and the interfaces created in the kinds (container and others). Thanks

**Comment by hellt** [comment, marked as answer]
Hi
maybe you are looking for `network-mode: none` - https://containerlab.dev/manual/nodes/#none-mode

**Reply by karampok** [reply]
that should work indeed. Of course a docker network clab with stay around in the machine without reason but is not important. Thanks
